src/Umbraco.Courier.Contrib.Resolvers/Umbraco.Courier.Contrib.Resolvers.csproj

```diff
@@ -75,9 +75,9 @@
       <HintPath>..\packages\Examine.0.1.57.2941\lib\Examine.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="FluentNHibernate, Version=1.2.0.712, Culture=neutral, PublicKeyToken=8aa435e3cb308880, processorArchitecture=MSIL">
-      <HintPath>..\packages\Courier.Core.2.52.15\lib\FluentNHibernate.dll</HintPath>
-      <Private>False</Private>
+    <Reference Include="FluentNHibernate, Version=1.4.0.0, Culture=neutral, PublicKeyToken=8aa435e3cb308880, processorArchitecture=MSIL">
+      <HintPath>..\packages\Courier.Core.3.1.0\lib\FluentNHibernate.dll</HintPath>
+      <Private>True</Private>
```

Comment:
I'd previously sent a PR to set "Copy Local" to false, so the dependency DLLs aren't copied over to the output /bin folder.
I think NuGet updates automatically set it back to "Copy Local" when re-referencing.

However the build script does only grab the "Umbraco.Courier.Contrib.Resolvers.dll" file, so it's not a show-stopper.
Guess I was looking for a _cleaner_ build. 😃